src/utils/elapsedTime.js

```diff
@@ -0,0 +1,16 @@
+const elapsedTime = (card) => {
+  const createdTime = new Date(card.createdAt || card.created_at);
+  const currentTime = new Date();
+  const timeDiff = Math.floor((currentTime - createdTime) / 1000 / 60);
+
+  if (timeDiff < 2) return "1 minute ago";
+  else if (timeDiff <= 59) return `${timeDiff} minutes ago`;
+  else if (timeDiff <= 60 * 23) return `${Math.floor(timeDiff / 60)} hours ago`;
+  else if (timeDiff <= 60 * 24 * 30)
+    return `${Math.floor(timeDiff / 24 / 60)} days ago`;
+  else if (timeDiff < 60 * 24 * 365)
+    return `${Math.floor(timeDiff / 30 / 24 / 60)} months ago`;
+  else return `${Math.floor(timeDiff / 30 / 24 / 60 / 12)} years ago`;
```

Comment:
요거는 else if 대신 얼리 리턴으로 리팩토링 가능 할 것 같아요.
```suggestion
  if (timeDiff < 2) return "1 minute ago";
  if (timeDiff <= 59) return `${timeDiff} minutes ago`;
  if (timeDiff <= 60 * 23) return `${Math.floor(timeDiff / 60)} hours ago`;
  if (timeDiff <= 60 * 24 * 30)
    return `${Math.floor(timeDiff / 24 / 60)} days ago`;
  if (timeDiff < 60 * 24 * 365)
    return `${Math.floor(timeDiff / 30 / 24 / 60)} months ago`;
  return `${Math.floor(timeDiff / 30 / 24 / 60 / 12)} years ago`;
```